utils/shared.js

```diff
@@ -478,6 +478,14 @@ const SSOConfig = {
     'UCM-SSO-p4f5m': ucmSSOConfig
 }
 
+const twilioSmsStatusTimeout = {
+    queued: 10000,
+    sending: 15000,
```

Comment:
These intermediate statuses ('queued', 'sending') will be overwritten, so we not bother to save them.